src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1210,4 +1210,12 @@ public static synchronized void updateBurns()
         FurnaceTileEntity.getBurnTimes().entrySet().forEach(e -> VANILLA_BURNS.put(e.getKey().delegate, e.getValue()));
     }
 
+    public static List<ItemStack> modifyLoot(List<ItemStack> list, LootContext context) {
+        LootModifierManager man = LootModifierManager.getInstance();
+        for(IGlobalLootModifier mod : man.getAllLootMods()) {
```

Comment:
This doesn't seem to address re-processing new list entries added by modifiers.
Such as re-running a smelting modifier after modifier that converts gravel to sand.
Is that something we want to care about at all?

Reply:
I don't know if we want to handle it or not. I'm sure someone will eventually have a use-case, but it was not in the list of things people cared about when I was writing the system. Such processing would have to cleanly handle probabalistic alterations (e.g. if gravel is converted to sand 10% of the time (first pass: fail) and the list gets re-processed, it should not attempt to recheck that element). I'm worried that this becomes non-trivial quickly.

Reply:
That is the entire issue of why I didn't work on this yet.
But, because things are being designed, and this is most definitely going to be a breaking change {processing entries, instead of the full list} it needs to be addressed.
Perhaps the processor should be `List<ItemStack> process(ItemStack);`

Reply:
I have a specific use-case that requires the full list, not just individual entries. One of the test mods (`WheatSeedsConverterModifier`) shows one such an example.